File: demo/springboot/demo/src/main/java/com/demo/domain/dto/http/resonse/OssUserBucketFileIdsResponse.java
package com.demo.domain.dto.http.resonse;

import lombok.Data;

import java.util.List;

@Data
public class OssUserBucketFileIdsResponse {
    private String userId;
    private String bucketName;
    /** File ids as decimal strings on the wire. */
    private List<String> fileIdList;
}

File: demo/kmp/shared/src/commonMain/kotlin/com/vectordemo/domain/dto/http/response/OssUserBucketFileIdsResponse.kt
package com.vectordemo.domain.dto.http.response

import kotlinx.serialization.Serializable

@Serializable
data class OssUserBucketFileIdsResponse(
    val userId: String? = null,
    val bucketName: String? = null,
    val fileIdList: List<String>? = null,
)
